Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:latest
+
+WORKDIR /file_hosting
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+EXPOSE 8000
+
+RUN go build -o main .
+
+CMD ./main
```

```suggestion
COPY . .

EXPOSE 8000

RUN go build -o main .

ENTRYPOINT ["/file_hosting/main"]
```

Лучше так. Зависимости проще хранить локально с помощью go mod vendor, и передавать при сборке в контейнер через COPY.